src/MinHashSimilarities.java

```diff
@@ -1,18 +1,96 @@
+import java.util.LinkedHashSet;
+import java.util.List;
+
 public class MinHashSimilarities {
+    int[][] tdMatrix;
+    int[][] mhMatrix;
+    LinkedHashSet<String> allTerms;
+    private List<String> indexedDocuments;
+    MinHash minhash;
+    double eJacSim;
+    double aJacSim;
+    int [] mhSig;
+    int numPerm;
+    int intersect = 0;
+    int union = 0;
+    int sameMH;
+
+
+    public MinHashSimilarities(String folder, int numPermutations) {
+        this.numPerm=numPermutations;
+
+        minhash = new MinHash(folder, numPermutations);
+        tdMatrix = minhash.termDocumentMatrix.getIntMatrix();
```

Comment:
You can call `minHash.termDocumentMatrix()` to get this (instead of `termDocumentMatrix.getIntMatrix()`). Under the hood the method `minHash.termDocumentMatrix()` calls `getIntMatrix()`